### PR TITLE
Chart service: settings available in Paper UI

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
Take a look what we already have for Basic UI and Classic UI: "Defines ...".
But I agree that it is not the same "pattern" everywhere.

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="ui:chart">
+		<parameter name="provider" type="text" required="false">
+			<label>Provider</label>
+			<description>Defines the chart provider to use.</description>
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: I wonder if we should have that displayed in the UI at all. There currently usually is only a single provider anyhow and it is rather a choice that is done by the solution and not so much by the user. We also do not have any mechanism to display available options to the user, so I would vote to leave that out (or at least to mark it advanced).

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			<default>240</default>
+		</parameter>
+		<parameter name="defaultWidth" type="text" required="false">
+			<label>Default Width</label>
+			<description>Defines the default width in pixels for the chart to build.</description>
+			<default>480</default>
+		</parameter>
+		<parameter name="scale" type="text" required="false">
+			<label>Scale</label>
+			<description>Defines the scale to apply to the requested chart height (0.5 to divide by 2 for example).</description>
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: Rather an advanced option, isn't it?

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			<default>1</default>
+		</parameter>
+		<parameter name="maxWidth" type="text" required="false">
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: Rather an advanced option, isn't it?

File: bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml [hunk 4]
```diff
@@ -17,6 +17,9 @@
    <implementation class="org.eclipse.smarthome.ui.internal.chart.ChartServlet"/>
    
    <property name="service.pid" type="String" value="org.eclipse.smarthome.chart"/>
+   <property name="service.config.description.uri" type="String" value="ui:chart"/>
+   <property name="service.config.label" type="String" value="Chart UI"/>
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml]: Hm, there isn't really such a thing as a "Chart UI" (compare to "Basic UI" etc.).

File: bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml [hunk 5]
```diff
@@ -17,6 +17,9 @@
...
+   <property name="service.config.category" type="String" value="ui"/>
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml]: The "ui" category is meant for UI extensions/addons. This here should imho rather be a "system" setting.

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml (hunk 2 above)
Comment (lolodomo) [reply bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: Ok

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml (hunk 3 above)
Comment (lolodomo) [reply bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: Ok

File: bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml (hunk 4 above)
Comment (lolodomo) [reply bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml]: "Chart" only ?

File: bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml (hunk 5 above)
Comment (lolodomo) [reply bundles/ui/org.eclipse.smarthome.ui/OSGI-INF/chartservlet.xml]: Ok

File: bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/ui/org.eclipse.smarthome.ui/ESH-INF/config/config.xml]: Ok I will suppress it.

**Pull request merged**